src/components/ModalView/ModalView.js

```diff
@@ -0,0 +1,272 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import styles from './ModalView.module.css'
+import { Modal, Image, Row, Col, Table, Button } from 'react-bootstrap';
+import ChartTool from '../ChartTool'
+import DobleChartTool from '../DobleChartTool'
+import { setShow } from '../../redux/actions/modalViewActions'
+import { cleanSelectedPokemons } from '../../redux/actions/pokemonsActions'
+import { setShowToast } from '../../redux/actions/pokemonsActions'
+
+const ModalView = ({
+  setShow,
+  modalViewState,
+  selectedPokemons,
+  cleanSelectedPokemons,
+  setShowToast,
+  showToast
+}) => {
+  const clases = styles
+
+  const handleClick = (e) => {
+    setShow(modalViewState)
+    cleanSelectedPokemons()
+    if (showToast) {
+      setShowToast()
+    }
+  }
+
+  const handleCompareClick = (e) => {
+    setShow(modalViewState)
+    setShowToast()
+  }
+
+  return (
+    <>
+      {showToast && selectedPokemons.length > 1
+        ?
+        // VIEW FOR 2 POKEMONS
```

Comment:
if comments are required, probably it is better to split this component into multiple smaller ones